### DISCUSSION TITLE
Feature Request: App Swipe left/right to scrub and double tap to pause play (for swipe  controls patch)

### DISCUSSION THREAD

**Discussion by 0-BlackSpectrum-0** [ReVanced/revanced-discussions#324] [Question] [answered]
It's a feature request, usually to scrub we've to long press first and then swipe and to pause we've to press on the button!

Feature:
If possible add this in swipe controls patch: >> swipe left and right to scrub through the video (like vlc/mx player) default action is to long press and scrub left and right which is not that intuitive and also add double tap in the middle area to pause and play too 

Why?:
The usual/default scrub is not that fast like to jump quickly 1s or 1min(precise variable scrubbing) ahead into the video also double tap to pause would be quite handy especially if there's text on screen and it question away quickly like subtitles etc double tap would be much quicker to pause and read!

You can check VLC for android it has all the features!

**Comment by TheJeterLP** [comment]
I like that you need to hold it first

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
This is a very old discussion, but there is now a swipe to seek patch that behaves similar to this.
